### ISSUE TITLE
Write a proper README

### ISSUE THREAD

**Issue by DivineOmega** [Continuum-Editor/Continuum-Editor#54]

**Comment by jbrooksuk** [comment]
Also make use of GitHub's Wiki, unless you can use Readme.io or something (the guys there are awesome btw).

**Comment by DivineOmega** [comment]
Agreed, I'm intending on using GitHub's wiki when I write docs for making Continuum addons (#47).

Readme.io looks like a really good service.

**Comment by jbrooksuk** [comment]
We use Readme.io for [Cachet's docs](https://docs.cachethq.io), the team there have always been supportive with Cachet and helping us out with some errors we had, worth looking at.

**Comment by DivineOmega** [comment]
Is the entire Cachet site Readme.io powered or only the docs subdomain?

**Comment by jbrooksuk** [comment]
Just the docs.
